src/components/landing/landing.css

```diff
@@ -1,73 +1,95 @@
 .landingBody {
-    background-color: #EEE2DC;
-    display: flex;
-    justify-content: center;
-    height: 100vh;
+  background-color: #EEE2DC;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+.landingLogoImg {
+  height: 60px;
+  padding: 10px;
+}
+
+.landingImg {
+  width: 350px;
 }
 
 .landingStyle {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-direction: column;
-    padding: 50px 20px;
-    margin-top: 5vh;
-    margin-bottom: 5vh;
-    border-radius: 50px;
-    background-color: white;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  border-radius: 50px;
+  background-color: #FFFFFF;
+  width: 450px;
+  padding: 60px 0;
 }
 
 .landingSpan {
-    display: flex;
-    padding: 20px;
+  display: flex;
+  padding: 20px;
 }
 
 .landingBtn {
-    border-radius: 6px;
-    padding: 15px;
-    width: 175px;
-    font-weight: bold;
-    text-align: center;
-    text-decoration: none;
-    font-size: 16px;
-    margin: 4px;
-    cursor: pointer;
-    transition: 0.3s;
-    cursor: pointer;
-  }
-  
-  .landingBtn:hover {
-    background-color: #ff725e;
-    color: black;
-  }
+  border-radius: 6px;
+  padding: 15px;
+  width: 175px;
+  font-weight: bold;
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  margin: 4px;
+  cursor: pointer;
```

Comment:
cursor: pointer; is listed here twice? Could that be impacting the click function?

Reply:
nah, that should not be the case, I think it's to do with React router and us using history etc

Reply:
Okay, are we removing the repeated cursor: pointer then?